test/common/upstream/load_balancer_benchmark.cc

```diff
@@ -163,16 +163,17 @@ void BM_MaglevLoadBalancerChooseHost(benchmark::State& state) {
     BaseTester tester(num_hosts);
     MaglevTable table(tester.priority_set_.getOrCreateHostSet(0).hosts());
     std::unordered_map<std::string, uint64_t> hit_counter;
-    TestLoadBalancerContext context;
     state.ResumeTiming();
 
     // Note: To a certain extent this is benchmarking the performance of xxhash as well as
     // std::unordered_map. However, it should be roughly equivalent to the work done when
     // comparing different hashing algorithms.
     for (uint64_t i = 0; i < keys_to_simulate; i++) {
-      context.hash_key_.value(
-          HashUtil::xxHash64(absl::string_view(reinterpret_cast<const char*>(&i), sizeof(i))));
-      hit_counter[table.chooseHost(&context)->address()->asString()] += 1;
+      hit_counter[table
+                      .chooseHost(HashUtil::xxHash64(
```

Comment:
this is pre-existing but recent :)

Maybe factor this hash-of-int hack into a helper.  It could even go into HashUtil.

Reply:
RE "new tests"; I thought your intent was to add new benchmarks ("I'm going to add more tests when I do the real LB implementation. Will merge this and then get to work on that.") but I don't know which ones you had in mind.  I don't know enough about the problem-space to suggest more.

AFAICT you already had covered benchmarks for init and operation pretty well.

Reply:
Sorry by new tests I meant all the tests I added in this PR about proper LB functioning within Envoy itself. I don't have any plans to add any more tests in a follow up, but happy to add more now if anyone has any suggestions.